lib/sis/sis.rb

```diff
@@ -51,13 +57,15 @@
 
 driver.find_element(:name => 'DERIVED_SAAWHIF_SSR_PB_SUBMIT').send_key("\n")
 
+puts 'generating report...'
 # wait for report to be finished
 wait.until { driver.find_element(:id => 'win0divDERIVED_SAA_DPR_GROUPBOX1$2').displayed? }
 driver.find_element(:name => 'DERIVED_SAA_DPR_SSS_EXPAND_ALL').click
 # wait for expansion to finish
 wait.until { driver.find_element(:id => 'WAIT_win0').displayed? }
 wait.until { !driver.find_element(:id => 'WAIT_win0').displayed? }
 
+puts 'parsing information...'
 major_element = driver.find_element(:id => 'win0divDERIVED_SAA_DPR_GROUPBOX1$2')
```

Comment:
`win0divDERIVED_SAA_DPR_GROUPBOX1$2 ` is the id for each of the purple table headers. I think we previously assumed there would be a constant number of purples, but Econ has 4 and I know CS only has 3. Need to find a way to make sure it's the major and not the general area requirements